src/electron/platform/android/scan-results-to-unified-results.ts

```diff
@@ -90,7 +90,10 @@ function createUnifiedResult(
         ruleId: ruleInformation.ruleId,
         status: getStatus(ruleResult.status),
         descriptors: getDescriptors(viewElementLookup[ruleResult.axeViewId]),
-        identifiers: null,
+        identifiers: {
+            identifier: viewElementLookup[ruleResult.axeViewId]?.className,
+            conciseName: viewElementLookup[ruleResult.axeViewId]?.className,
```

Comment:
Are we ok with both of these as duplicate values or would it be better to set one blank/null?

Reply:
The reason I'm using both is later in bug filing we expect both, in unified-result-to-issue-fling-data.ts line 29, 30, otherwise it will create null error